### PR TITLE
HPCC-18836 DOCS: Updates to VM doc

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-18836 Updates to VM doc
Update the VM doc to match the steps in the
Virtual Box application.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review

successful build tests and resulting PDF can be obtained from http://10.240.32.243/job/DocDev06/

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Successful Build test and resulting PDF can be viewed at:
http://10.240.32.243/job/DocDev06/

### CODE REVIEW

File: docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml [hunk 1]
```diff
@@ -231,65 +231,6 @@
...
-            <para>From the <emphasis role="bold">File</emphasis> menu, select
-            <emphasis role="bold">Preferences</emphasis>...</para>
-
-            <para>The <emphasis>VirtualBox - Settings</emphasis> dialog
-            displays.<figure id="VBoxSettings">
-                <title xreflabel="welc">VirtualBox - Settings</title>
-
-                <mediaobject>
-                  <imageobject>
-                    <imagedata fileref="images/vmimg14.jpg" />
```
Comment (JamesDeFabia) [inline docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml]: If image is no longer used, the jpg needs to be deleted

File: docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml [hunk 2]
```diff
@@ -231,65 +231,6 @@
...
-          <listitem>
-            <para>From the<emphasis> Network Details</emphasis> window, select
-            the <emphasis role="bold">DHCP Server</emphasis> Tab and check the
-            <emphasis role="bold">Enable Server</emphasis> box. <figure
-                id="NetworkDetails">
-                <title xreflabel="welc">Network Details</title>
-
-                <mediaobject>
-                  <imageobject>
-                    <imagedata fileref="images/vmimg15.jpg" />
```
Comment (JamesDeFabia) [inline docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml]: If image is no longer used, the jpg needs to be deleted

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
See comments inline

File: docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml (hunk 2 above)
Comment (g-pan) [reply docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml]: good point. Deleted images.

**Comment by g-pan** [comment]
@kunalaswani can you give this a quick review (for content) as well.  Note that these are the steps in running HPCC on a Virtual Box, the revised PDF (Running HPCC in a Virtual Machine) can be found on http://10.240.32.243/job/DocDev06/  (pages 7-8)

**Comment by kunalaswani** [comment]
@g-pan the change you've made is accurate, and those steps all look correct. Thanks

**Pull request merged**